domains/kappucino.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "kappucin",
+           "email": "132572179+kappucin@users.noreply.github.com",
+           "discord": "853170099392479253"
+        },
+    
+        "record": {
+            "CNAME": "kappucin.github.io"
```

Comment:
CNAME redirects back to this domain which you are trying to register which is an invalid record. No content is shown.